addons/website/static/src/scss/website.wysiwyg.scss

```diff
@@ -209,9 +209,24 @@ $i: 1;
 
 .pac-container { // google map autosuggestion
     z-index: $zindex-modal-backdrop; // > $o-we-zindex
-    min-width: 400px;
+    width: $o-we-sidebar-width !important;
+    font-size: 12px;
+    margin-left: -$o-we-sidebar-width/2;
```

Comment:
doesnt look right, looks like it works by chance... If you change, for example, the width of the we-title's (input labels)
You will see that it doesnt work ;)

Reply:
There is no other solution except this as of now. Because Google provides UI element so we can just add style. So in my opinion rather than giving fix random numbers to width or random calculation, we can use at least a near-by solution.

rest of the thing has been improved. if still there is room for improvement please let me know. Thanks!